src/app/[locale]/layout.tsx

```diff
@@ -12,6 +12,7 @@ import 'bootstrap/dist/css/bootstrap.min.css'
 import '@/styles/globals.css'
 import '@/styles/auth.css'
 import '@/styles/gridjs/sw360.css'
+import 'react-toastify/dist/ReactToastify.css'
```

Comment:
I like Toastify, but for the sake of consistency, can you look if we have something in react-bootstrap ?